website/docs/r/spring_cloud_config_server.html.markdown

```diff
@@ -0,0 +1,109 @@
+subcategory: "App Platform"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud_config_server"
+sidebar_current: "docs-azurerm-resource-spring-cloud-config-server"
+description: |-
+  Manage Azure Spring Cloud Config Server instance.
+---
+
+# azurerm_spring_cloud
+
+Manage Azure Cloud Config Server instance. Spring Cloud Config provides server and client-side support for an externalized configuration in a distributed system. With the Config Server instance, you have a central place to manage external properties for applications across all environments.
+
+Azure Spring Cloud Config Server supports Azure DevOps, GitHub, GitLab, and Bitbucket for storing your Config Server files
+
+Azure Spring Cloud Config Server supports Public repository, Private repository with SSH authentication or Private repository with basic authentication
+
+Azure Spring Cloud Config Server also supports multiple repositories with Pattern Matching
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "resGroup"
+  location = "Southeast Asia"
+}
+
+resource "azurerm_spring_cloud" "example" {
+  name                     = "sc-example"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+
+  tags = {
+    environment = "staging"
+  }
+}
+
+resource "azurerm_spring_cloud_config_server" "example" {
+  label                    = "config"
+  spring_cloud_id          = "azurerm_spring_cloud.example.id
+  uri                      = "https://github.com/Azure-Samples/piggymetrics"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:

```

Comment:
COuld we make all this documentation consistent with the rest of the provideR?

values should be `block quoted`: `true`, `1`, `https://`